gradoop-flink/src/main/java/org/gradoop/flink/algorithms/gelly/hits/HITSToAttributes.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright © 2014 - 2017 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.algorithms.gelly.hits;
+
+import org.apache.flink.api.common.functions.JoinFunction;
+import org.apache.flink.graph.library.link_analysis.HITS;
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.common.model.impl.properties.PropertyValue;
+
+/**
+ * Stores HITS Results as Properties of a Vertex
+ */
+public class HITSToAttributes implements JoinFunction<HITS.Result<GradoopId>, Vertex, Vertex> {
```

Comment:
This file needs to be moved into a sub-package called `functions` (see pagerank or connectedcomponents)